test/dhcp/server_isc_test.rb

```diff
@@ -82,8 +82,24 @@ def test_poap_quirks
   def test_loadSubnets_loads_managed_subnets
     subnets = @dhcp.loadSubnets
```

Comment:
Nods,that's correct. However, currently these tests verify the behaviour of `parse_config_for_subnets` method: so either calls to `loadSubnets`should be replaced with calls to `parse_config_for_subnets`, or the assertions should be made against `Proxy::DHCP::SubnetService` (the main side-effect of `loadSubnets` is initialization of `SubnetService`).